src/_playground/documentation/Properties/_PropertyShape.js

```diff
@@ -7,31 +8,36 @@ import React from 'react';
 import sortBy from 'sort-by';
 import Table from '../../../Table/Table';
 
-const PropertyShape = ({ title, propTypes, defaultProps, propDescriptions }) => {
+const PropertyShape = ({ title, propTypes = {}, defaultProps = {}, propDescriptions = {} }) => {
     if (!propTypes) {
         return (
             <em>This shape has no defined properties.</em>
         );
     }
 
     let propInfo = Object.keys(propTypes).map((propName) => {
-        return { propName: propName, required: propTypes[propName].typeRequired };
+        return { propName: propName, required: !!propTypes[propName].typeRequired };
```

Comment:
Force a boolean to prevent sorting issues (if `typeRequired` was undefined rather than false, props were sorting at the bottom for what appeared to be no reason).